test/lib.test.js

```diff
@@ -7944,6 +8449,188 @@ describe('[orderbook]', () => {
   });
 });
 
+test.only('orderbook.fetchRequesterOrderbook() strict', async () => {
```

Comment:
remove `test.only` + typo in method name

```suggestion
test('orderbook.fetchRequestOrderbook() strict', async () => {
```